proofs/refine.ml

```diff
@@ -47,6 +47,7 @@ let typecheck_evar ev env sigma =
 let generic_refine ~typecheck f gl =
   let sigma = Proofview.Goal.sigma gl in
   let env = Proofview.Goal.env gl in
+  if !Flags.debug then Feedback.msg_debug Pp.(str"sigma before generic_refine: " ++ Termops.pr_evar_map None env sigma);
```

Comment:
It seems to me that this debug could would make the `-debug` option unusable, right?